### PR TITLE
docs: add docs of create table like parquet

### PR DESCRIPTION
**Pull request by tobegit3hub**
Add docs of create table like parquet in Chinese and English.

### CODE REVIEW

**Review by dl239 — APPROVED** [review]

File: docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md [hunk 1]
```diff
@@ -18,18 +18,20 @@ TableElement ::=
...
 ```sql
 CreateTableStmt ::=
-    'CREATE' 'TABLE' TableName LIKE 'HIVE' PATH
+    'CREATE' 'TABLE' TableName LIKE LikeType PATH
 
 TableName ::=
     Identifier ('.' Identifier)?
 
+LikeType ::=
+    ('HIVE' 'PARQUET')?
```
Comment (zhanghaohit) [inline docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md]:
```
LikeType ::=
    'HIVE' | 'PARQUET'
```

File: docs/zh/openmldb_sql/ddl/CREATE_TABLE_STATEMENT.md [hunk 2]
```diff
@@ -22,7 +22,23 @@ TableElement ::=
...
+
+```sql
+CreateTableStmt ::=
+    'CREATE' 'TABLE' TableName LIKE LikeType PATH
+
+TableName ::=
+    Identifier ('.' Identifier)?
+
+LikeType ::=
+    ('HIVE' 'PARQUET')?
```
Comment (zhanghaohit) [inline docs/zh/openmldb_sql/ddl/CREATE_TABLE_STATEMENT.md]:
```
LikeType ::=
    'HIVE' | 'PARQUET'
```

File: docs/zh/openmldb_sql/ddl/CREATE_TABLE_STATEMENT.md [hunk 3]
```diff
@@ -184,6 +200,15 @@ CREATE TABLE db1.t1 LIKE HIVE 'hive://hive_db.t1';
...
 注意目前创建的表名必须包括DB名。
 
+**示例6：基于 Parquet 文件创建新表**
+
+```sql
+CREATE TABLE db1.t1 LIKE PARQUET 'file://t1.parquet';
+-- SUCCEED
+```
+
+注意目前创建的表名必须包括DB名。
```
Comment (zhanghaohit) [inline docs/zh/openmldb_sql/ddl/CREATE_TABLE_STATEMENT.md:210]: db is not optional if we have `use db`?

Comment (tobegit3hub) [reply docs/zh/openmldb_sql/ddl/CREATE_TABLE_STATEMENT.md:210]: It is a bug now and we will fix in https://github.com/4paradigm/OpenMLDB/pull/3063 .

**Review by zhanghaohit — APPROVED** [review]

**Pull request merged**